docs/management/index.md

```diff
@@ -1233,6 +1234,83 @@ DELETE /api/reset
 rabbitmqctl eval 'rabbit_mgmt_storage:reset_all().'
 ```
 
+## Troubleshooting {#troubleshooting}
+
+### OpenId Discovery endpoint not reachable {#openid-Discovery-endpoint-not-reachable}
```

Comment:
The common practice is to use lowercase characters only.

Reply:
i think in this case, this is the name of the specification which everywhere it is written in camel-case .e.g. https://swagger.io/docs/specification/authentication/openid-connect-discovery/. I can try to use lower case ...